en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md

```diff
@@ -33,11 +33,11 @@ In Choreo Connect, version 3.0.x series the open API definition required 2 manda
 
 `x-wso2-basePath` is used to group all HTTP resources of the OpenAPI under a single context. Therefore, it will be exposed as a sub resource of a single API using Choreo Connect. This is how Choreo Connect logically separates one API from the other. The `x-wso2-production-endpoints` refers to the actual backend service implementation of the API. Therefore, the latter mentioned details are mandatory to expose an API using Choreo Connect.
 
-**But with 3.1.0 onwards none of the WSO2 specific open API extensions are mandatory.** Choreo Connect resolves base path and back end URL as below
+**But with 3.1.0 onwards none of the WSO2 specific open API extensions are mandatory.** Choreo Connect resolves base path and back end URL as follows:
```

Comment:
Let's remove the below line. It is used in v310 Ballerina mgw.

But with 3.1.0 onwards none of the WSO2 specific open API extensions are mandatory

Reply:
fixed in https://github.com/wso2/docs-apim/pull/5032/commits/f02f25aa5ccc060afcd096f91bb1273a3fc042a8